lib/HttpClient/CurlClient.php

```diff
@@ -38,6 +38,9 @@ public static function instance()
 
     protected $defaultOptions;
 
+    /** @var \Stripe\Util\RandomGenerator */
+    protected $randomGenerator;
+
```

Comment:
Can you say more about this change?

Reply:
We're just declaring a class property that we were already using. PHP doesn't force you to declare class properties, but it's good practice to do so.